test/Distributed/actor_protocols.swift

```diff
@@ -14,8 +14,9 @@ typealias DefaultDistributedActorSystem = FakeActorSystem
 actor A: Actor {} // ok
 
 class C: Actor, UnsafeSendable {
-  // expected-error@-1{{non-actor type 'C' cannot conform to the 'Actor' protocol}} {{1-6=actor}}
-  // expected-warning@-2{{'UnsafeSendable' is deprecated: Use @unchecked Sendable instead}}
+  // expected-error@-1{{non-actor type 'C' cannot conform to the 'AnyActor' protocol}} {{1-6=actor}}
+  // expected-error@-2{{non-actor type 'C' cannot conform to the 'Actor' protocol}} {{1-6=actor}}
```

Comment:
Aww. I think we can check whether a conformance was implied, so we don't diagnose multiple times.

Reply:
That'd be good; any hint onto how easily? Or I have to walk the exact protocols from ast which are inherited or something?